### PR TITLE
Add features to changelog

### PR DESCRIPTION
**Pull request by timbeiko**
Signed-off-by: Tim Beiko <t.beiko23@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 21.1.1
+
+## 21.1.1 Features
+
+Features added between from 20.10.0 to 21.1.1 include:
```
Comment (macfarla) [inline CHANGELOG.md:7]: s/between from/from

Comment (macfarla) [reply CHANGELOG.md:7]: or between? idk which is better

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+* Besu Launcher: Besu now has support for the [Quorum Mainnet Launcher](https://github.com/ConsenSys/quorum-mainnet-launcher) which makes it easy for users to configure and launch Besu on the Ethereum mainnet.
+* Bonsai Tries (early access): A new database format which reduces storage requirements and improves performance for access to recent state. _Note: only full sync is currently supported._
+* Miner Data JSON RPC: The `eth_getMinerDataByBlockHash` and `eth_getMinerDataByBlockNumber` endpoints return miner rewards and coinbase address for a given block.
```
Comment (macfarla) [inline CHANGELOG.md:10]: s/JSON RPC/JSON-RPC

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+* Bonsai Tries (early access): A new database format which reduces storage requirements and improves performance for access to recent state. _Note: only full sync is currently supported._
+* Miner Data JSON RPC: The `eth_getMinerDataByBlockHash` and `eth_getMinerDataByBlockNumber` endpoints return miner rewards and coinbase address for a given block. 
+* EIP-1898 support: [The EIP](https://eips.ethereum.org/EIPS/eip-1898) adds `blockHash` to JSON-RPC methods which accept a default block parameter.
+
+### Additions and Improvements
+* Added `besu_transaction_pool_transactions` to the reported metrics, counting the mempool size [\#1869](https://github.com/hyperledger/besu/pull/1869)
+
+### Bug Fixes
+
+### Early Access Features
```
Comment (macfarla) [inline CHANGELOG.md:18]: should bonsai trees be here?

**Review by macfarla — COMMENTED** [review]
couple of comments

**Comment by timbeiko** [comment]
Thanks for the comments, @macfarla. Addressed in the latest commit!

**Review by macfarla — APPROVED** [review]
LGTM

**Pull request merged**